src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -292,6 +394,36 @@ public function getSameSite()
         return $this->sameSite;
     }
 
+    /**
+     * Creates a cookie copy with SameSite attribute.
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function withSameSite(?string $sameSite = 'lax'): self
```

Comment:
the argument should be mandatory and not nullable

Reply:
for now, `$this->sameSite` is expected to be null here https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Cookie.php#L176. There's also a line that transforms an empty string into `null` here https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Cookie.php#L122.